octoprint_octorelay/__init__.py

```diff
@@ -216,70 +224,71 @@ def print_stopped(self):
                 self.turn_off_timers[index] = ResettableTimer(
-                    autoOffDelay, self.turn_off_pin, [relay_pin, inverted, cmdOFF])
+                    delay, self.turn_off_pin, [relay_pin, inverted, cmd_off])
                 self.turn_off_timers[index].start()
         self.update_ui()
 
-    def turn_off_pin(self, relay_pin: int, inverted: bool, cmdOFF):
+    def turn_off_pin(self, relay_pin: int, inverted: bool, cmd_off):
         GPIO.setup(relay_pin, GPIO.OUT)
         # XOR with inverted
         GPIO.output(relay_pin, inverted is not False)
         GPIO.setwarnings(True)
-        if cmdOFF:
-            os.system(cmdOFF)
+        if cmd_off:
+            os.system(cmd_off)
         self._logger.info(f"pin: {relay_pin} turned off")
         self.update_ui()
 
-    def turn_on_pin(self, relay_pin: int, inverted: bool, cmdON):
+    def turn_on_pin(self, relay_pin: int, inverted: bool, cmd_on):
         GPIO.setup(relay_pin, GPIO.OUT)
         # XOR with inverted
         GPIO.output(relay_pin, inverted is False)
         GPIO.setwarnings(True)
-        if cmdON:
-            os.system(cmdON)
+        if cmd_on:
+            os.system(cmd_on)
         self._logger.info(f"pin: {relay_pin} turned on")
 
     def update_ui(self):
         settings = DEFAULT_SETTINGS.copy()
         for index in RELAY_INDEXES:
             settings[index].update(self._settings.get([index]))
 
-            labelText = settings[index]["labelText"]
+            label_text = settings[index]["labelText"]
             active = int(settings[index]["active"]) # todo make it bool later
             relay_pin = int(settings[index]["relay_pin"])
             inverted = bool(settings[index]['inverted_output'])
-            iconOn = settings[index]['iconOn']
-            iconOff = settings[index]['iconOff']
-            confirmOff = bool(settings[index]['confirmOff'])
+            icon_on = settings[index]['iconOn']
+            icon_off = settings[index]['iconOff']
+            confirm_off = bool(settings[index]['confirmOff'])
 
             # set the icon state
             GPIO.setup(relay_pin, GPIO.OUT)
             self.model[index]['relay_pin'] = relay_pin
             self.model[index]['state'] = GPIO.input(relay_pin) # int
-            self.model[index]['labelText'] = labelText
+            self.model[index]['labelText'] = label_text
             self.model[index]['active'] = active
             if inverted is not bool(self.model[index]['state']):
-                self.model[index]['iconText'] = iconOn
-                self.model[index]['confirmOff'] = confirmOff
+                self.model[index]['iconText'] = icon_on
+                self.model[index]['confirmOff'] = confirm_off
             else:
-                self.model[index]['iconText'] = iconOff
+                self.model[index]['iconText'] = icon_off
                 self.model[index]['confirmOff'] = False
```

Comment:
whole this block can be simplified.
it has too many redundant back-and-forward assignments